### DISCUSSION TITLE
bugifx ? #6409

### DISCUSSION THREAD

**Discussion by uRhaineWork** [netbox-community/netbox#9327] [Q&A] [answered]
Hi, I would like to ask if issue from
https://github.com/netbox-community/netbox/discussions/6409
really is resolved ? Im encountering exact problem:
1. Add 2 devices (stack)
2. Add chassis with unit 1 as primary
3. Add 10g interface to unit 1
4. add 10g interface to unit 2
5. add LAG interface to unit 1
6. PROBLEM -> cant add interface 10g interface under unit 2 to LAG on master chassis unit 1. I can add 10G from unit 1, on other/multiple from this same unit.

Python 3.9.9 / NB 3.2.0

**Comment by DanSheps** [comment, marked as answer]
This will be in the next release
